pkg/exporter/direct_flp.go

```diff
@@ -0,0 +1,44 @@
+package exporter
+
+import (
+	"fmt"
+
+	flpconfig "github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/ingest"
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/flow"
+	"gopkg.in/yaml.v2"
+)
+
+// DirectFLP flow exporter
+type DirectFLP struct {
+	*ingest.InProcess
+}
+
+func StartDirectFLP(jsonConfig string) (*DirectFLP, error) {
+	var cfg flpconfig.ConfigFileStruct
+	// Note that, despite jsonConfig being json, we use yaml unmarshaler because the json one
+	// is screwed up for HTTPClientConfig in github.com/prometheus/common/config (used for Loki)
+	// This is ok as YAML is a superset of JSON.
+	// E.g. try unmarshaling `{"clientConfig":{"authorization":{}}}` as a api.WriteLoki
+	// See also https://github.com/prometheus/prometheus/issues/11816
```

Comment:
interesting thanks for referring to the issue in promo its almost 1yr so probably this work around will stay for sometime

Reply:
Yeah there's even a PR from @OlivierCazade that is older than that :-) https://github.com/prometheus/common/pull/394